independent-publisher-connectors/AfterShip/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2690 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "AfterShip",
+    "description": "AfterShip - Shipment tracking API"
+  },
+  "host": "api.aftership.com",
+  "basePath": "/v4",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/couriers": {
+      "get": {
+        "summary": "Get user activated couriers",
+        "description": "Return a list of [couriers activated at your AfterShip account](https://admin.aftership.com/settings/couriers).",
+        "operationId": "GetUserActivatedCouriers",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "description": "Content-Type",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "meta": {
+                  "type": "object",
+                  "properties": {
+                    "code": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "code"
+                    }
+                  },
+                  "description": "meta"
+                },
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "total": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "total"
+                    },
+                    "couriers": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "slug": {
+                            "type": "string",
+                            "description": "slug"
+                          },
+                          "name": {
+                            "type": "string",
+                            "description": "name"
+                          },
+                          "phone": {
+                            "type": "string",
+                            "description": "phone"
+                          },
+                          "other_name": {
+                            "type": "string",
+                            "description": "other_name"
+                          },
+                          "web_url": {
+                            "type": "string",
+                            "description": "web_url"
```

Comment:
@taiki-yoshida All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
@taiki-yashida I can not see changes to the description values.